llama_stack/providers/inline/agents/meta_reference/openai_responses.py

```diff
@@ -229,7 +234,9 @@ async def create_openai_response(
 
         input = await self._prepend_previous_response(input, previous_response_id)
         messages = await _convert_response_input_to_chat_messages(input)
+        await self._prepend_instructions(messages, instructions)
```

Comment:
why prepend? why not start `messages` with instruction first and _append_ later?

Reply:
`messages` comes out of `_convert_response_input_to_chat_messages`, to start it with `instructions` I would need to pass `instructions` into `_convert_response_input_to_chat_messages` and start it in there, I can do this it just didn't seem relevant to the functions purpose.